aws-cs-langserve/Aws.Langserve.csproj

```diff
@@ -2,7 +2,7 @@
 
 	<PropertyGroup>
 		<OutputType>Exe</OutputType>
-		<TargetFramework>net6.0</TargetFramework>
+		<TargetFramework>net8.0</TargetFramework>
```

Comment:
Curious, does this have to be .NET 8? Can it be 6? Asking because we still support 6 (and run 6 in CI), so if possible we keep to that if we can.

Reply:
```suggestion
		<TargetFramework>net6.0</TargetFramework>
```

Reply:
@cnunciato ups! This is my setup but you do not need 8.0 to run it! Thanks for the feedback. I fixed it